go.mod

```diff
@@ -9,20 +9,25 @@ require (
 	github.com/edgexfoundry/go-mod-core-contracts v0.1.25
 	github.com/edgexfoundry/go-mod-messaging v0.1.11
 	github.com/edgexfoundry/go-mod-registry v0.1.11
-	github.com/edgexfoundry/go-mod-secrets v0.0.7
+	github.com/edgexfoundry/go-mod-secrets v0.0.14
 	github.com/golang/snappy v0.0.1 // indirect
 	github.com/gomodule/redigo v2.0.0+incompatible
 	github.com/google/go-cmp v0.3.1 // indirect
 	github.com/google/uuid v1.1.0
+	github.com/gorilla/context v1.1.1 // indirect
 	github.com/gorilla/mux v1.7.2
+	github.com/kr/pretty v0.2.0 // indirect
 	github.com/pelletier/go-toml v1.2.0
-	github.com/stretchr/objx v0.1.1 // indirect
-	github.com/stretchr/testify v1.3.0
+	github.com/stretchr/objx v0.2.0 // indirect
+	github.com/stretchr/testify v1.4.0
 	github.com/tidwall/pretty v1.0.0 // indirect
 	github.com/ugorji/go v1.1.4
 	github.com/xdg/scram v0.0.0-20180814205039-7eeb5667e42c // indirect
 	github.com/xdg/stringprep v1.0.0 // indirect
 	go.mongodb.org/mongo-driver v1.1.1
 	golang.org/x/text v0.3.2 // indirect
-	gopkg.in/yaml.v2 v2.2.2 // indirect
+	gopkg.in/check.v1 v1.0.0-20190902080502-41f04d3bba15 // indirect
+	gopkg.in/yaml.v2 v2.2.8 // indirect
 )
+
+replace github.com/edgexfoundry/go-mod-secrets => ../go-mod-secrets
```

Comment:
What do you need from your local reference to go-mod-secrets?  
The 'replace' should be removed and if you do need a change in go-mod-secrets that change should be a new PR for go-mod-secrets.

Reply:
The local replacement should be removed once we are trying to merge into the edgex's master branch. For now it is ok.

Reply:
Curious why you needed local copy?

Reply:
used it for debug. will remove